components/automate-ui/src/app/modules/desktop/empty-data/empty-data.component.html

```diff
@@ -0,0 +1,2 @@
+<img class="empty-icon" src="assets/img/chart.svg" width="50" alt="" aria-hidden="true" />
+<p class="empty-message">Data not found. Try checking in your Chef Infra Client configuration and give us a moment to load it.</p>
```

Comment:
The phrase "checking in" seems... challenging. First I thought it was "checking... inside your config", but then I thought, no, it is probably "checking-in" almost like a git commit, but pretty sure that is not the intent. So I am not at all clear what it is asking the user to do. Could this be rephrased?

Reply:
@msorens I don't have a strong opinion at the moment; I just grabbed the wording from the mockup. I'm certainly open to changing it, adding link(s) to docs, etc. since I'm sure some users could find this ambiguous.

Reply:
kimberly is out til monday, so ill open a card to address the rewording and put it in docs chat and merge this so it can be done!